specification/maps/data-plane/Route/preview/2024-04-01-preview/route.json

```diff
@@ -0,0 +1,2097 @@
+        "operationId": "Route_SubmitDirectionsAsynchronousBatch",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Submit an asynchronous batch job for directions": {
+            "$ref": "./examples/DirectionsAsyncBatch.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/stable/2023-06-01/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          },
+          {
+            "name": "directionsAsyncBatchRequestBody",
+            "in": "body",
+            "description": "The request body for directions asynchronous batch request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AsyncBatchRequestBody"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Operation-Location": {
+                "description": "URL to check the status of the asynchronous operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code of the error that occurred."
+              }
+            },
+            "x-ms-error-response": true
+          }
+        }
```

Comment:
Do you have a final schema after the LRO (e.g. which container in the storage account contains the results)?

If yes, please add `final-state-schema` refer to it.
https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-long-running-operation-options

Reply:
added